.circleci/config.yml

```
@@ -744,10 +744,14 @@ jobs:
 
   smoke-tests:
     parameters:
+      parallelism:
+          type: integer
+          default: 8  # this should coorespond with the resource-class defined in the executor
```

nit: typo (also on lines 769 and 795) 
```suggestion
          default: 8  # this should correspond with the resource-class defined in the executor
```

---

🤦🏻‍♀️ I will fix before merging. ty!